content/03.methods.md

```diff
@@ -3,20 +3,50 @@
-PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
 
-#### Nucleic acids extraction and library preparation
+#### Children's Brain Tumor Tissue Consortium (CBTTC)
 
-##### PNOC samples
+The CBTTC [@url:https://cbttc.org] is a collaborative, multi-institutional (16 institutions worldwide) research program dedicated to the study of childhood brain tumors. 
+All CBTTC data can be download from the Gabriella Miller Kids First Data Resource Center (KF-DRC), [@url:https://kidsfirstdrc.org].
+The deidentified patient's blood and tumor tissue were prospectively collected by the consortium from patients enrolled within the CBTTC. 
+
+The cell lines were generated by the CBTTC from either fresh tumor tissue obtained firectly from surgery performed at Children’s Hospital of Philadelphia (CHOP) or from prospectively collected tumor specimens stored in Recover Cell Culture Freezing media (cat# 12648010, Gibco).
+The tissue was dissociated using enzymatic method with papain as described [@doi:10.1101/656587].
+Briefly, tissue was washed with HBSS (cat# 14175095, Gibco), minced and icubated with activated papin solution (cat# LS003124, SciQuest) for up to 45 minutes. 
+The papain was inactivated using ovomucoid solution (cat# 542000, SciQuest), tissue was briefly treated with DNase (cat#  10104159001, Sigma) and passed through the 100μm cell strainer (cat# 542000, Greiner Bio-One). 
+Two cell culture conditions were initiated based on the number of cells avialble. 
+For cultures utilizing the fetal bovine serum (FBS), a minimum density of 3×10^5 cells/ml were plated in DMEM/F-12 medium (cat# D8062, Sigma) supplemented with 20% FBS (cat# SH30910.03, Hyclone), 1% GlutaMAX (cat# 35050061, Gibco), Penicillin/Streptomycin-Amphotericin B Mixture (cat# 17-745E, Lonza) and 0.2% Normocin (cat# ant-nr-2, Invivogen). 
+For the serum-free media conditions cells were plated at minimum density of 1×10^6 cells/ml in DMEM/F12 media supplemented with 1% GlutaMAX, 1x B-27 supplement minus vitamin A (cat# 12587-010, Gibco), 1x N-2 supplement (cat# 17502001, Gibco), 20 ng/ml epidermal growth factor (cat# PHG0311L, Gibco), 20 ng/ml basic fibroblast growth factor (cat# 100-18B, PeproTech), 2.5μg/ml heparin (cat# H3149, Sigma), Penicillin/Streptomycin-Amphotericin B Mixture and 0.2% Normocin.
+
+#### Pacific Pediatric Neuro Onclology Consortium (PNOC)
+
+The Pacific Pediatric Neuro-Oncology Consortium (PNOC) is an international consortium dedicated to bringing new therapies to children and young adults with brain tumors.
+PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](@url:https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
```

Comment:
```suggestion
PNOC collected blood and tumor biospecimens from newly-diagnosed DIPG patients as part of the clinical trial [PNOC003/NCT02274987](https://clinicaltrials.gov/ct2/show/NCT02274987) [@doi:10.1002/ijc.32258]. 
```

This should be as it was because it is an in-line style link and not a citation. I will make this change. It's the last thing to do before merging.